### PR TITLE
mp-alert

### PR DESCRIPTION
**Pull request by rickydoar**
mp-alert modal

### CODE REVIEW

**Comment by tdumitrescu** [comment]
this should be an `mp-modal` IMO

**Comment by jbwyme** [comment]
@tdumitrescu why do you think that?

**Comment by tdumitrescu** [comment]
Because it doesn't seem any more different from the other 4 or so existing modal types than they are from each other. But I don't have a lot of details, so don't feel real strongly.

File: lib/components/alert/index.jade [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- const proxyAttrs = {'modal-type': 'popup'};
+- ['closeable', 'open', 'width'].forEach(attr => proxyAttrs[attr] = $component.getAttribute(attr));
```
Comment (jbwyme) [inline lib/components/alert/index.jade]: no need to proxy `width` anymore, I've made it so you can specify it using the css. Make sure you're up to date with the latest master and take a look at the `mp-confirm` example: https://github.com/mixpanel/mixpanel-common/blob/master/lib/components/confirm/index.styl#L8

**Pull request merged**